spot-client/src/spot-tv/ui/components/meeting-frame/MeetingFrame.js

```diff
@@ -18,6 +28,30 @@ export default class MeetingFrame extends Component {
      * @returns {ReactElement}
      */
     render() {
-        return <JitsiMeetingFrame { ...this.props } />;
+        const {
+            allowZoomMeetings,
+            ...otherProps
+        } = this.props;
+
+        if (allowZoomMeetings && isZoomMeetingUrl(otherProps.meetingUrl)) {
```

Comment:
We were writing about this on chat, but I think this should not fallback to jitsi meeting if detected as zoom URL, but fail to join the meeting instead.

Reply:
Maybe that check for zoom enabled can happen in `redirectToMeeting` action where are other URL safety checks?

Reply:
and here it could just select based on URL type without the `allowZoomMeetings` check

Reply:
it may be easier to fire failed to join from the frame though, the redirect action has no rejection handlers

Reply:
Should we do the whitelist we talked about instead of targeting zoom specifically?

Reply:
Do you mean the whitelist would include zoom URL if it's enabled?

Reply:
Yeah, so the configuration would need both the zoom integration configured plus the whitelist.